data-reset-tool/pre-hook-update-db.sh

```diff
@@ -0,0 +1 @@
+# THIS IS EMPTY - but present so that deployment config is consistent with other python applications.
```

Comment:
I think we should delete this file since it's not being used and is a bit confusing if you don't know that